internal/commands/whoami/command.go

```diff
@@ -10,7 +12,7 @@ import (
 var CommandMeta = cli.CommandMeta{
 	Use:         "whoami",
 	Description: "Display information about the current user",
-	// TODO(REALMC-8832): this is an example of where standardizing cli, comamnd and flag names
+	// TODO(REALMC-8832): this is an example of where standardizing cli, command and flag names
 	// into a shared package would be helpful, to reduce coupling the command packages to each other
 	// (since this HelptText creates a "whoami depends on login" package cycle)
```

Comment:
missed one...
```suggestion
	// (since this HelpText creates a "whoami depends on login" package cycle)
```

on a more serious note, this ticket has since been worked on and closed, so we should figure out what to do here.  the intent of this comment was remove the potential for this help text to ever get stale, if we possibly changed the `login` command name to something else

while it feels like overkill, it's also not wrong.  can we file a new ticket under the `BAAS-13516` epic:

`Create centralized file for all CLI command and flag names`

and in the description, just add that this file could/should probably live in the `cli/cli.go` package file.  also, we should update this TODO with the new ticket number

Reply:
https://jira.mongodb.org/browse/BAAS-14174